content/getting-started/setup/telerik-removal/index.md

```diff
@@ -81,17 +81,21 @@ Navigate to `Settings > Extensions (Modules)` in the `Persona Bar` and uninstall
 Navigate to `Settings > Extensions (Providers)` in the `Persona Bar` and uninstall the `RadEditor Manager` extension.  Be sure to check the `Delete Files` checkbox.  NOTE: Not all installations will have this extension installed.
 
 ### Step 15
+Navigate to `Settings > Extensions (Providers)` in the `Persona Bar` and uninstall any `Telerik` related extension(s).  Be sure to check the `Delete Files` checkbox.  NOTE: Not all installations will have these extension installed.
```

Comment:
This seems a bit vague and generalized.  We should provide specific extension name(s) here.  Thoughts?

Reply:
You are right it's a bit vague. My idea was that there could be other Telerik editor extensions in there, but I guess that not very likely.